lib/index.js

```diff
@@ -8,6 +8,12 @@
  * @return {Date} The cloned date object.
  */
 module.exports = function cloneDate(date) {
+    // prevent error message in case of an invalid entry
+    try {
```

Comment:
Same here, throw `TypeError` if `date` is not a `Date`.

Reply:
Sorry, I was wanting to help, as they both had the same mistake, I sent them to help